src/attestation/Attester.chain.spec.ts

```diff
@@ -57,14 +57,14 @@ describe('Test Attester on chain', () => {
     }).addFromMnemonic(mnemonic)
     expect(attesterAddress).toBe(fromMiniSecret)
     expect(attesterAddress).toBe(fromMnemonic)
-    expect(attesterAddress).not.toBe(fromSeed)
+    expect(attesterAddress).toBe(fromSeed)
     expect(attesterAddress).toBe(attesterAddressFromURI)
   }, 10000)
   // NOTE: if this test fails after updating @polkadot packages, we should adjust Attester.chain.ts
   // see issue https://github.com/polkadot-js/common/blob/d889c71056158df72b34b994506d062c2e731cc0/packages/keyring/src/keyring.ts#L174
   it('Should generate correct keypair for signature type ed25519', async () => {
     const mnemonic = AttesterChain.generateMnemonic()
-    const seed = mnemonicToSeed(mnemonic)
```

Comment:
Couldn't find it in the logs, but `mnemonicToSeed` [was removed](https://github.com/polkadot-js/common/blob/master/packages/keyring/src/keyring.ts#L172). Probably similar to the removal of `toSeed` in [v2.12.1](https://github.com/polkadot-js/common/releases/tag/v2.12.1), see below.